diagnostic.md

```diff
@@ -73,14 +89,26 @@ data in JSON format.
 ```
 
 ```sh
-# your answer here
+curl -H --request --include "Content-Type: application/json" POST {
```

Comment:
Should look like this:

`curl --include --request "POST" "http://example.com/donuts" \
  --header "Content-Type: application/json" \
  --data $'{
    "donut": {
      "name": "French Cruller",
      "price": "$0.99"
    }
  }'`